.github/workflows/main.yml

```diff
@@ -47,7 +47,7 @@ jobs:
           ocaml-compiler: ocaml.5.0.0,ocaml-option-mingw
           opam-repositories: |
             dra27: https://github.com/dra27/opam-repository.git#windows-5.0
```

Comment:
How far does it get if we remove the dra27 repo here?

Reply:
Not far I believe, because:

 - We can't create a 5.0.0 compiler switch
 - We need the patches to ocamlfind I think too https://github.com/dra27/opam-repository/tree/windows-5.0/packages/ocamlfind/ocamlfind.1.9.5/files

That is, maybe, all we're getting out of that repo.

Reply:
Thanks, I'll look at merging the ocamlfind one tomorrow.  We can get by without a compiler switch with an ocaml-system install possibly.